src/js/components/DateInput/DateInput.js

```diff
@@ -62,6 +64,20 @@ const DateInput = forwardRef(
       initialValue: defaultValue,
     });
 
+    let timestamp;
```

Comment:
Can this be merged with the similar code in Calendar.js around line 274?

Reply:
I was running into errors where when I separated it out into its own function and assigned `timestamp` by calling the function, it wasn't handling this properly. That's why I've kept in inline here.